src/nvim/api/autocmd.c

```diff
@@ -74,6 +74,7 @@ static int64_t next_autocmd_id = 1;
 ///             Cannot be used with {buffer}
 ///             - buffer: Buffer number or list of buffer numbers for buffer local autocommands
 ///             |autocmd-buflocal|. Cannot be used with {pattern}
+///             - id: (integer|array) autocmd ids
```

Comment:
Is there a strong need to support array? If someone has an id they usually just want 1 result.

Reply:
keeping the code simple (no array support) is probably worth it for now

Reply:
okay 👍